ush/setup.py

```diff
@@ -540,6 +540,95 @@ def remove_tag(tasks, tag):
         post_meta = rocoto_tasks.get("metatask_run_ens_post", {})
         post_meta.pop("metatask_run_sub_hourly_post", None)
         post_meta.pop("metatask_sub_hourly_last_hour_post", None)
+    #
+    # -----------------------------------------------------------------------
+    #
+    # Remove all verification [meta]tasks for which no fields are specified.
+    #
+    # -----------------------------------------------------------------------
+    #
+    vx_fields_all = {}
+    vx_metatasks_all = {}
+
+    vx_fields_all["CCPA"] = ["APCP"]
+    vx_metatasks_all["CCPA"] = ["metatask_PcpCombine_obs", 
+                                "metatask_PcpCombine_fcst_all_accums_all_mems", 
+                                "metatask_GridStat_CCPA_all_accums_all_mems",
+                                "metatask_GEPES_CCPA",
+                                "metatask_GridStat_CCPA_ensmeanprob_all_accums"]
+
+    vx_fields_all["MRMS"] = ["REFC", "RETOP"]
+    vx_metatasks_all["MRMS"] = ["metatask_GridStat_MRMS_all_mems",
+                                "metatask_GEPES_MRMS",
+                                "metatask_GridStat_MRMS_ensprob"]
+
+    vx_fields_all["NDAS"] = ["SFC", "UPA"]
+    vx_metatasks_all["NDAS"] = ["task_run_MET_Pb2nc_obs",
+                                "metatask_PointStat_NDAS_all_mems",
+                                "metatask_GEPES_NDAS",
+                                "metatask_PointStat_NDAS_ensmeanprob"]
+
+    vx_fields_config = expt_config["verification"]["VX_FIELDS"]
+    for obstype in vx_fields_all:
+        vx_fields_obstype = [field for field in vx_fields_config if field in vx_fields_all[obstype]]
+        if not vx_fields_obstype:
+            for metatask in vx_metatasks_all[obstype]:
+                if metatask in rocoto_config['tasks']:
+                    logging.info(dedent(
+                        f"""
+                        Removing verification [meta]task
+                          "{metatask}"
+                        from workflow since no fields belonging to observation type "{obstype}"
+                        are specified for verification."""
+                    ))
+                    rocoto_config['tasks'].pop(metatask, None)
```

Comment:
The logic below here would not be needed if the switches are removed as suggested above.

Reply:
@christinaholtNOAA Yep!  Will remove as part of next set of changes.

Reply:
@christinaholtNOAA Removed.